Quotes.js

```diff
@@ -1,38 +1,16 @@
 import React, { useState, useEffect } from 'react';
 import { StyleSheet, Text, TouchableOpacity, View } from 'react-native';
-import { useIsShake } from './useIsShake';
-import { QUOTES_IN_DURATION, QUOTES_IN_DELAY, QUOTES_OUT_DURATION, SVG_OUT_DURATION, SVG_IN_DURATION, SVG_IN_DELAY, quotes } from './constants';
+import { quotes } from './constants';
 import { getRandomInt } from './utils';
 
 export default function Quotes(props) {
-  const [quote, setQuote] = useState(0);
-  const [registered, setRegistered] = useState(false);
-  const [toggledBefore, setToggledBefore] = useState(false);
-  const { toggle } = useIsShake();
+  const { quote } = props.quote;
+  const { setQuote } = props.setQuote;
```

Comment:
Props isolated as values to separate keys for consistent naming conventions.

Reply:
This will be updated when you update the call to `Quotes` in other components